config.toml

```diff
@@ -0,0 +1,329 @@
+koji_target = 'f39-pythonxx'
+
+[distgit]
+branch = "rawhide"
+commit_message = "Rebuilt for Python3.12"
+bootstrap_commit_message = "Bootstrap for Python 3.11"
+author = "Python Maint <python-maint@redhat.com>"
+
+[architectures]
+repoquery =  "x86_64" # used for repository querying
+koji = "x86_64" # used to scratch build packages with bconds
+
+[deps]
+old = ["python(abi) = 3.11", "libpython3.11.so.1.0()(64bit)", "libpython3.11d.so.1.0()(64bit)"]
+new = ["python(abi) = 3.12", "libpython3.12.so.1.0()(64bit)", "libpython3.12d.so.1.0()(64bit)"]
+
+[components]
+excluded = ["python3.11", "python3.12"]
+extra = ["python3-docs"]
+
+[cache_dir]
+dnf = "_dnf_cache_dir"
+fedpkg = "_fedpkg_cache_dir"
+
+[repos]
+[[repos.rawhide]]
+repoid = "rawhide"
+# metalink = "https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=$basearch"
+baseurl = ["http://kojipkgs.fedoraproject.org/repos/rawhide/latest/$basearch/"]
+
+[[repos.rawhide]]
+repoid = "rawhide-source"
+# metalink = "https://mirrors.fedoraproject.org/metalink?repo=rawhide-source&arch=$basearch"
+baseurl = ["http://kojipkgs.fedoraproject.org/repos/rawhide/latest/src/"]
+
+[[repos.target]]
+repoid = "python3.12"
+baseurl = ["https://copr-be.cloud.fedoraproject.org/results/@python/python3.12/fedora-rawhide-$basearch/"]
+#baseurl = ["http://kojipkgs.fedoraproject.org/repos/f39-python/latest/$basearch/"]
+metadata_expire = 60
+
+[bconds]
+[[bconds.python-setuptools]]
+withs = ["bootstrap"]
+withouts = ["tests"]
+
+[[bconds.pyparsing]]
+withs = ["bootstrap"]
+
+[[bconds.python-packaging]]
+withs = ["bootstrap"]
+withouts = ["tests", "docs"]
+
+[[bconds.python-wheel]]
+withs = ["bootstrap"]
+
+[[bconds.python-pip]]
+withouts = ["tests", "doc"]
+
+[[bconds.python-six]]
+withouts = ["tests"]
+
+[[bconds.python-toml]]
+withouts = ["tests"]
+
+[[bconds.python-tomli]]
+withs = ["bootstrap"]
```

Comment:
I've removed those in the meantime. Sorry about that.

Please rebase this and use the up-to-date list of bconds. Thanks.

I'll stop touching the bconds list for now.